stable/pipelines/values.yaml

```diff
@@ -2,7 +2,7 @@
 
 ## Common
 initContainer:
-  image: "alpine:3.11"
+  image: "alpine:3.12"
```

Comment:
@rimusz  can you change it to `docker.bintray.io/alpine:3.12`

Reply:
it already has it https://github.com/jfrog/charts/blob/pipelines-chart-v1.4.0/stable/pipelines/values.yaml#L5

Reply:
done